semeio/workflows/ahm_analysis/ahmanalysis.py

```diff
@@ -423,20 +394,23 @@ def calc_delta_grid(
     return mygrid_ok_short
 
 
+def _get_parameter_keys(parameters, prior_keys):
+    transfp_keys = []
+    for parameter in parameters:
+        for key in prior_keys:
+            if "_" + parameter + ":" in key:
+                transfp_keys.append(key)
+    return set(prior_keys).difference(transfp_keys)
+
+
 def get_updated_parameters(prior_data, parameters):
     """make list of updated parameters
     (excluding duplicate transformed parameters)
     """
-    transfp_keys = []
-    p_keys = []
-    p_keysf = []
-    for paramet in parameters:
-        for key in prior_data.keys():
-            if "_" + paramet + ":" in key:
-                transfp_keys.append(key)
```

Comment:
What does `transfp` mean?

Reply:
this is to collect all the parameters which are transformed (ert can generate a new parameter named for instance LOG10_GENKWID:PARAM in addition to GENKWID:PARAM ) and therefore give the option to plot the transformed or not transformed distribution later in webviz.

Reply:
Thanks for the clarification, would it be possible to add slightly more descriptive variable names here?

Reply:
changed to something more explicit.